omnibus/files/private-chef-cookbooks/private-chef/libraries/du.rb

```diff
@@ -0,0 +1,25 @@
+require 'mixlib/shellout'
+module Du
+  # Calculate the disk space used by the given path. Requires that
+  # `du` is in our PATH.
+  #
+  # @param path [String] Path to a directory on disk
+  # @return [Integer] KB used by directory on disk
+  #
+  def self.du(path)
+    # TODO(ssd) 2017-08-18: Do we need to worry about sparse files
+    # here? If so, can we expect the --apparent-size flag to exist on
+    # all of our platforms.
+    command = Mixlib::ShellOut.new("du -sk #{path}")
```

Comment:
How about `du -P`?

[this sounds compelling](https://en.wikipedia.org/wiki/Df_(Unix)#Usage):
> -P
>   Use a standard, portable, output format